mission_1.css

```
@@ -0,0 +1,262 @@
+*{
+  box-sizing: border-box;
+  margin:0;
+  padding:0;
+}
+header{
+  width: 1920px;
+  height: 70px;
+  padding: 10px 200px 9px 200px;
+  gap: 0px;
+  border: 0px 0px 1px 0px;
+  opacity: 0px;
+  background: #FFFFFF;
+  border-bottom: 1px solid #DFDFDF;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+.logo{
+  margin: auto;
+  cursor: pointer;
+}
+.login{
+  width: Fixed (128px)px;
+  height: Fixed (48px)px;
+  top: 127px;
+  left: 538px;
+  padding: 12px 20px 12px 20px;
```

상하 값이 같고 좌우 값이 같으면 아래와 같이 축약형으로 작성할 수 있답니다~!
```suggestion
  padding: 12px 20px ;
```